### PR TITLE
84 recurring event

### LINKED ISSUE #84: As a user, I should be able to add recurring timing for event.
So that I don't need to create new event every time.

### PR DESCRIPTION
**Pull request by tjonganthony**
Fixes #84 

Main Changes:

- New field in Event class, recursion, which indicates milliseconds between each recurrence.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -39,16 +39,19 @@
...
      */
-    public AddCommand(String title, LocalDateTime startTime, LocalDateTime endTime, Set<String> tags, String description)
+    public AddCommand(String title, LocalDateTime startTime, LocalDateTime endTime, Set<String> tags, String description, long recursion)
             throws IllegalValueException, IllegalArgumentException {
         final Set<Tag> tagSet = new HashSet<>();
         for (String tagName : tags) {
             tagSet.add(new Tag(tagName));
         }
 
+        System.out.println("repeat = " + recursion);
```
Comment (bertoia) [inline src/main/java/seedu/address/logic/commands/AddCommand.java]: println

Comment (tjonganthony) [reply src/main/java/seedu/address/logic/commands/AddCommand.java]: fixed

File: .settings/org.eclipse.buildship.core.prefs [hunk 2]
```diff
@@ -1,7 +1,6 @@
 build.commands=org.eclipse.jdt.core.javabuilder
 connection.arguments=
 connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
-connection.gradle.user.home=null
```
Comment (bertoia) [inline .settings/org.eclipse.buildship.core.prefs:4]: Is this line causing gradle build error? If not can add it back?

**Review by bertoia — APPROVED** [review]
Looks good to go.

**Pull request merged**